core/src/main/java/org/apache/iceberg/SnapshotProducer.java

```diff
@@ -499,21 +500,34 @@ protected OutputFile manifestListPath() {
                         "snap-%d-%d-%s", snapshotId(), attempt.incrementAndGet(), commitUUID))));
   }
 
+  /**
+   * @deprecated since 1.6.0 . Will be removed in 1.7.0. Use {@link
```

Comment:
I think we usually add in the deprecation notes right before release (I don't think there will be a 1.6 or 1.7

Reply:
I think we do want to add these right away. If we don't have a 1.7 then we'll rewrite to "removed in 2.0"